W-16334397 cloud connector alt text - A (#2658)
mulesoft/docs-connectors: amazon-dynamodb/1.5/modules/ROOT/pages/amazon-dynamodb-connector-studio.adoc

The following screenshot shows an example of configuring Basic authentication:

.Basic authentication fields

> image::amazon-dynamodb-basic-authentication.png["Configure authentication by selecting Basic in the Connection field and then complete the fields on the General tab"]

Author: ```suggestion
image::amazon-dynamodb-basic-authentication.png["Basic option selected in the Connection field"]
```

The following screenshot shows an example of configuring Role authentication:

.Role authentication fields

> image::amazon-dynamodb-role-authentication.png["Configure Role authentication by selecting Role in the Connection field and then complete the fields on the General tab"]

Author: ```suggestion
image::amazon-dynamodb-role-authentication.png["Role option selected in the Connection field"]
```
